release-prep.sh

```diff
@@ -0,0 +1,211 @@
+#!/bin/bash
+
+# This script will do code related release preparation stuff for Bitcoin Core as specified in
+# the release-process.md file.
+# This should be run from the folder containing the Source tree
+# The following actions will be done:
+#  1. Set the version numbers as specified in the arguments
+#  2. Update src/chainparams.cpp nMinimumChainWork and defaultAssumeValid with information from the getblockchaininfo rpc.
+#  3. Update Hard coded seeds
+#  4. Set BLOCK_CHAIN_SIZE
+#  5. Update translations
+#  6. Generate updated manpages
+# Note: Step 2 assumes that an up-to-date Bitcoin Core is running and has been built in the
+# directory which this script is being run.
+
+# Variables
+VERSION=
+BLOCK_CHAIN_SIZE=
+DATADIR=""
+verBump=true
+chainparamsUpdate=true
+seedUpdate=true
+blockchainsizeBump=true
+translationsUpdate=true
+genManpages=true
+
+# Help Message
+read -d '' usage <<- EOF
+Usage: $scriptName version block_chain_size
+
+Run this script from the Bitcoin Core Source root directory. This requires a current version of Bitcoin Core
+to be running at the time that this script is run.
+
+Arguments:
+version		    Version number to set following the MAJOR.MINOR.REVISION format. Only required if
```

Comment:
Nit: Would avoid `\t`'s (or be consistent and not mix with space) if possible. (This looked weirdly indented in git.) Mostly seem to use tabs. Could maybe s&r `\t` -> ` `.